client/web/src/hooks/node-data.ts

```diff
@@ -20,6 +16,12 @@ export type NodeData = {
   IPNVersion: string
 }
 
+export type UserProfile = {
+  LoginName: string
+  DisplayName: string
+  ProfilePicURL: string
+}
+
 // testData is static set of nodedata used during development.
 // This can be removed once we have a real node data API.
 const testData: NodeData = {
```

Comment:
I think we can probably remove the testData?

Reply:
whoop! yes!